src/editor/Containers/Editor.tsx

```diff
@@ -0,0 +1,29 @@
+import React, {useMemo} from 'react';
+
+import {PageConstructor} from '../../containers/PageConstructor';
+import AddBlock from '../Components/AddBlock/AddBlock';
+import EditBlock from '../Components/EditBlock/EditBlock';
+import {EditorContext} from '../context';
+import {useEditorState} from '../store';
+import {EditBlockConstructorProps, EditorProps} from '../types';
+
+export const Editor = ({data: initialData, custom}: EditorProps) => {
```

Comment:
it is not very clear what a `custom` is. Maybe that's how to call this variable `customConfig`?

Reply:
I made refactoring here, but renaming of custom requires a major change which I tried to avoid in this pr.
Maybe custom should be renamed in `customExеensions` 🤔

Reply:
customExtensions is nice!